Src/FluentAssertions/Types/TypeSelectorAssertions.cs

```diff
@@ -477,5 +477,5 @@ private static string GetDescriptionFor(Type type)
 
     /// <inheritdoc/>
     public override bool Equals(object obj) =>
-        throw new NotSupportedException("Calling Equals on Assertion classes is not supported.");
+        throw new NotSupportedException("Equals is not part of Fluent Assertions. Did you mean Be() instead?");
```

Comment:
🔧 This class doesn't have a `Be`. Maybe mention `BeInNamespace` and `BeDecoratedWith`?